packages/cozy-client/src/store/queries.js

```diff
@@ -139,15 +139,28 @@ export const convert$gtNullSelectors = selector => {
   return result
 }
 
+/**
+ * Merges query selectors with query partial indexes
+ *
+ * @param {object} queryDefinition - A query definition
+ * @returns {object} A query definition selector
+ */
+export const mergeSelectorAndPartialIndex = queryDefinition => ({
+  ...queryDefinition.selector,
+  ...get(queryDefinition, 'partialFilter')
+})
+
 const getSelectorFilterFn = queryDefinition => {
+  console.info('qdef', queryDefinition)
```

Comment:
I guess this should removed ;)

Reply:
https://github.com/cozy/cozy-client/pull/849